### ISSUE TITLE
`list packages` command fails in version 0.35.0

### ISSUE THREAD

**Issue by mikewallace1979** [cloudsmith-io/cloudsmith-cli#91]
The following command fails since upgrading to cloudsmith-cli version 0.35.0:

```
$ cloudsmith list packages --output-format json ORG/REPO
Getting list of packages ... OK
Failed to convert to JSON: Object of type datetime is not JSON serializable
```

The same command is successful with cloudsmith-cli version 0.34.0.

I re-ran with `-v` and `-d` which resulted in the following log line:

    2022-12-14 16:20:56,608 DEBUG https://api.cloudsmith.io:443 "GET /v1/packages/ORG/REPO/?page=1&page_size=30 HTTP/1.1" 200 None

So I guess it's still using the v1 API and failing since it now only supports the v2 API.

**Comment by paddycarey** [comment]
Thanks for the report @mikewallace1979, the team is looking at this right now

**Comment by jackgibson1** [comment]
Hey @mikewallace1979. This issue should now be fixed in `v0.35.1`. Thanks for letting us know!

**Comment by mikewallace1979** [comment]
No problem - can confirm it is indeed fixed!
